src/__tests__/MongoMemoryServer-test.ts

```diff
@@ -0,0 +1,91 @@
+import MongoMemoryServer from '../MongoMemoryServer';
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 600000;
+
+describe('MongoMemoryServer', () => {
```

Comment:
@nodkz I've added this test suite to cover a bit more the functions of MongoMemoryServer class